lib/bundle/brew_dumper.rb

```diff
@@ -17,12 +17,10 @@ def self.formulae
 
     def self.dump
       formulae.map do |f|
-        if f[:args].empty?
-          "brew '#{f[:full_name]}'"
-        else
-          args = f[:args].map { |arg| "'#{arg}'" }.sort.join(", ")
-          "brew '#{f[:full_name]}', args: [#{args}]"
-        end
+        brewline = "brew '#{f[:full_name]}'"
+        args = f[:args].map { |arg| "'#{arg}'" }.sort.join(", ")
+        brewline += ", args: [#{args}]" unless f[:args].empty?
+        brewline += BrewServices.started?(f[:full_name]) ? ", service_restart: true" : ''
```

Comment:
Wed Aug 10 08:22:05 BST 2016

Tried that originally.  It produces blank lines apart from the ones that have 'service_restart'.